DeviceTests/Caboodle.DeviceTests.Shared/Utils.cs

```diff
@@ -1,10 +1,14 @@
 ﻿using System;
 using System.Threading.Tasks;
+using Microsoft.Caboodle;
 
 namespace Caboodle.DeviceTests
 {
     public class Utils
     {
+        public static bool IsiOSSimulator
```

Comment:
Could this be a method that we pass in the device platform?

Reply:
This was more just for the tests, as the iOS sim was quite different

Reply:
ahhh gotcha <3 I see it is devicetests cool.